wrappers/android/examples/native_example/app/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set_target_properties(
     realsense2
     PROPERTIES
     IMPORTED_LOCATION
-    ${CMAKE_CURRENT_SOURCE_DIR}/build/jni/${ANDROID_ABI}/librealsense2.so
+    ${CMAKE_CURRENT_SOURCE_DIR}/../../../build/jniLibs/${ANDROID_ABI}/librealsense2.so
```

Comment:
Having to hardcode the build directory name seems broken.  Please use `${CMAKE_CURRENT_BINARY_DIR}` instead.

Reply:
native_example is not part of android's wrapper project which produces .so files. From the scope of the example, ${CMAKE_CURRENT_BINARY_DIR} refers to location in project's file hierarchy which is located in different location than libraries output

Reply:
I see.  Thanks.